pkg/accesscontrol/policy_rule_index.go

```diff
@@ -20,13 +19,12 @@ type policyRuleIndex struct {
 	rCache              v1.RoleCache
 	crbCache            v1.ClusterRoleBindingCache
 	rbCache             v1.RoleBindingCache
-	revisions           *roleRevisionIndex
```

Comment:
I moved this to AccessStore, since it was only used by `addRolesToHash` which also makes more sense there. `policyRuleIndex` does not need to know about any hashing.